### PR TITLE
RHDEVDOCS 5729 overriding a task in remote pipeline

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

cp to pipelines-docs-1.13

Issue:

RHDEVDOCS 5729

Link to docs preview:

https://68447--docspreview.netlify.app/openshift-pipelines/latest/pac/using-pac-resolver#overriding-tasks-in-remote-pipeline_using-pac-resolver

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc [hunk 1]
```diff
@@ -21,3 +21,25 @@ You can share a pipeline definition across multiple repositories by using the re
 You can reference only one pipeline definition using the annotation.
 ====
 
+[id="overriding-tasks-in-remote-pipeline_{context}"]
+== Overriding a task in a remote pipeline
+
+By default, if you use a remote pipeline annotation in a pipeline run, {pac} uses all the tasks that are a part of the remote pipeline.
+
+To override a task, you can add an annotation for a task with the same name to the remote pipeline annotation.
```
Comment (piyush-garg) [inline modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc:29]: nit: task with the same name as in remote pipeline to the pipelinerun annotation

**Comment by piyush-garg** [comment]
lgtm

**Review by ppitonak — APPROVED** [review]

File: modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc [hunk 2]
```diff
@@ -21,3 +21,25 @@ You can share a pipeline definition across multiple repositories by using the re
 You can reference only one pipeline definition using the annotation.
 ====
 
+[id="overriding-tasks-in-remote-pipeline_{context}"]
+== Overriding a task in a remote pipeline
+
+By default, if you use a remote pipeline annotation in a pipeline run, {pac} uses all the tasks that are a part of the remote pipeline.
+
+To override a task, you can add an annotation for a task with the same name as a task in the pipeline that the remote pipeline annotation references.
```
Comment (mburke5678) [inline modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc:29]: This line is not clear to me, but I am not familiar with pipelines. Maybe something like:
```suggestion
You can override a task in a remote pipeline by adding a task that has the same name as an annotation in the pipeline run.
```

File: modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc [hunk 3]
```diff
@@ -21,3 +21,25 @@ You can share a pipeline definition across multiple repositories by using the re
...
+By default, if you use a remote pipeline annotation in a pipeline run, {pac} uses all the tasks that are a part of the remote pipeline.
+
+You can override a task in a remote pipeline by adding a task annotation to the pipeline run. The added task must have the same name as a task in the remote pipeline.
+
+For example, you might use the following pipeline run definition:
+
+.Example pipeline run definition referencing a remote pipeline and overriding a task
+[source,yaml]
+----
+kind: PipelineRun
```
Comment (bergerhoffer) [inline modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc:36]: Should this have an apiVersion here?

File: modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc [hunk 4]
```diff
@@ -21,3 +21,25 @@ You can share a pipeline definition across multiple repositories by using the re
...
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/pipeline: "https://git.provider/raw/pipeline.yaml"
+    pipelinesascode.tekton.dev/task: "./my-git-clone-task.yaml"
+----
+
+For this example, assume the remote task found at \https://git.provider/raw/pipeline.yaml includes a task named `git-clone` and the task that the `my-git-clone-task.yaml` file defines is also named `git-clone`.
```
Comment (bergerhoffer) [inline modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc:43]: the URL here should probably be in backticks

**Review by bergerhoffer — COMMENTED** [review]
Added two questions/comments, but not holding the merge since it was QE reviewed and the other comment is minor.

But please consider checking these and making a future update if one is needed

**Pull request merged**